packs/feats/forced-entry.json

```diff
@@ -37,6 +37,16 @@
                 "selector": "athletics",
                 "slug": "no-crowbar",
                 "suppress": true
+            },
```

Comment:
Consider adding a link to `@UUID[Compendium.pf2e.actionspf2e.Item.Force Open]` in the description above.